layouts/partials/blocks/carousel.html

```diff
@@ -0,0 +1,522 @@
+<div id="carousel-container" class="carousel-container pt-16">
```

Comment:
When I opened the page, it took a second for the image to load, so it looked wrong against a white background. Maybe give put a light gray background under the carousel for before it loads?